pyface/ui/qt4/util/event_loop_helper.py

```diff
@@ -111,9 +111,14 @@ def event_loop_until_condition(self, condition, timeout=10.0):
             `timeout` is rounded to the nearest millisecond.
         """
 
+        condition_result = False
```

Comment:
It may be useful to go one step further here and set `condition_result = None` initially. That'll make it easier when debugging to detect the case when the condition wasn't tested at all.

Reply:
Not worth it.